PetAdoptions/cdk/pet_stack/resources/destroy_stack.sh

```diff
@@ -22,13 +22,14 @@ STACK_NAME_APP=$(aws ssm get-parameter --name '/eks/petsite/stackname' --region
 
 # Set default name in case Parameters are gone (partial deletion)
 if [ -z $STACK_NAME ]; then STACK_NAME="Services"; fi
-if [ -z $STACK_NAME_APP ]; then STACK_NAME_APP="Applications"; fi 
+if [ -z $STACK_NAME_APP ]; then STACK_NAME_APP="Applications"; fi
+if [ -z $STACK_NAME_CODEPIPELINE ]; then STACK_NAME_CODEPIPELINE="Observability-Workshop"; fi
 
 # Fix for CDK teardown issues
 aws eks update-kubeconfig --name PetSite
 kubectl delete -f https://raw.githubusercontent.com/aws-samples/one-observability-demo/main/PetAdoptions/cdk/pet_stack/resources/load_balancer/crds.yaml
 
-#Deleting keycloak 
+#Deleting keycloak
 kubectl delete namespace keycloak --force
 
 # Get rid of all resources (Application first, then cluster or it will fail)
```

Comment:
We need to check of the file `cdk.json` exists in the CWD before running these command (`cdk destroy`) as it will fail with the following error:

`--app is required either in command-line, in cdk.json or in ~/.cdk.json`

Reply:
Added a validation at the top of the script. In the cleanup instructions we should also precise where and how the script should be invoked

Reply:
But this validation will prevent the script to run on cloud shell since we're not cloning the repo (different MR) so the folder doesn't exist.

Reply:
Removed cdk destroy all together. Not sure I fully understand what you want though

Reply:
Basically to run cdk destroy only if `cdk.json` file exists, otherwise skip it